### DISCUSSION TITLE
获取suite_access_token的类有误

### DISCUSSION THREAD

**Discussion by AiKangLin-lab** [w7corp/easywechat#2542] [Q&A]
$suiteAccessToken = new SuiteAccessToken($config['suite_id'], $config['suite_secret']);

Composer:
    public function __construct(
        protected string $suiteId,
        protected string $suiteSecret,
        protected SuiteTicketInterface $suiteTicket,
        protected ?string $key = null,
        ?CacheInterface $cache = null,
        ?HttpClientInterface $httpClient = null,
    ) {
        $this->httpClient = $httpClient ?? new Client();
        $this->cache = $cache ?? new Psr16Cache(new FilesystemAdapter(namespace: 'easywechat', defaultLifetime: 1500));
    }

需要实现一个SuiteTicketInterface  但是文档并没有给出

**Comment by overtrue** [comment]
已优化：https://github.com/w7corp/easywechat/commit/5aeb35bab843c58729907c1d1cda1d8864d15cfb
